packages/config-cli/README.md

```diff
@@ -23,3 +23,11 @@ Dump the screenshots with config file
 ./bin/bmc.js screenshot --config s3://..../config.json.gz
 
 ```
+
+## Usage -- Bundle
+
+Bundle config files into config bundle json from a given config path.
+
+```bash
+./bin/bmc.js screenshot --config config/ --bundle config/config.json
```

Comment:
`bundle` ? 

bundling into the output directory could be interesting, maybe make the recommendation just `--output config.json` ?

Reply:
Updated.